### PR TITLE
Remove use of deprecated Mix.Config

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: test/test_helper.exs [hunk 1]
```diff
@@ -5,6 +5,7 @@ Code.require_file("test/support/test_filter.exs")
 Code.require_file("test/support/test_gen_server.exs")
 Code.require_file("test/support/test_error_view.exs")
 
-ExUnit.start()
+ExUnit.start(assert_receive_timeout: 500)
```
Comment (whatyouhide) [inline test/test_helper.exs:8]: Bonus thing to remove some config 🙃

**Comment by yordis** [comment]
Do you need all those config files, btw? 

It is a lib so probably only a `test` environment is needed here (or just `config/config.exs`, no?

**Comment by whatyouhide** [comment]
@yordis yes, that's next step 👍 Unless you wanna get on a PR, I'd be happy to have some help 😄

**Comment by yordis** [comment]
@whatyouhide count me on it!

**Pull request merged**